lib/sms.js

```diff
@@ -1,5 +1,33 @@
 const ph = require('./plugin-helper')
 const argv = require('minimist')(process.argv.slice(2))
+const _ = require('lodash/fp')
+
+const customSms = require('./custom-sms')
+
+const getDefaultMessageContent = content => ({
+  smsCode: `Your cryptomat code: ${content.code}`,
+  cashOutDispenseReady: `Your cash is waiting! Go to the Cryptomat and press Redeem within 24 hours. [${content.timestamp}]`
+})
+
+function getSms (event, phone, content) {
+  return customSms.getCustomMessage(event)
+    .then(msg => {
+      if (!_.isNil(msg)) {
+        var accMsg = msg.message
+        const contentKeys = _.keys(content)
+        const messageContent = _.reduce((acc, it) => _.replace(`#${it}`, content[it], acc), accMsg, contentKeys)
```

Comment:
Will this work for different languages? What if we are inserting numbers and we have to adjust plurals?